src/services/subscriber/subscriber.service.go

```diff
@@ -0,0 +1,19 @@
+package SubscriberService
+
+type Subscriber struct {
+	Webhook		string
+}
+
+// Implementing a pseudo observer pattern
+
+var subscribers []*Subscriber
```

Comment:
In the future turn the webhook into a channel so we can add subscribers on the fly?

Reply:
Hmm im wondering, since here `subscribers` is a pointer to Subscriber list, do we need to make it into a channel? i mean the pointer value doesnt change throughout the lifetime of the service, only the list it points to can be updated. In which case our polling subroutine will always keep the same pointer (memory address) and we dont need to make it into a channel. This is my understanding, lmk if that makes sense, im not sure lol, i dont know enough about Go.